spec/acceptance/trusted_external_data_spec.rb

```diff
@@ -38,6 +29,17 @@
   end
 
   shared_context 'setup cmdb' do |cmdb_table = 'cmdb_ci', certname_field = 'fqdn'|
+    before(:all) do
```

Comment:
Why move this here?

Reply:
I don't think this is where it will end up. I wanted to closer to the tests that actually need it, but when placed here it executes too many times.

Reply:
Ok, thanks